tests/common/platform/device_utils.py

```diff
@@ -0,0 +1,23 @@
+"""
+Helper script for fanout switch operations
+"""
+
+def fanout_switch_port_lookup(fanout_switches, dut_port):
+    """
+        look up the fanout switch instance and the fanout switch port
+        connecting to the dut_port
+
+        Args:
+            fanout_switches (list FanoutHost): list of fanout switch
+                                               instances.
+            dut_port (str): port name on the DUT
+
+        Returns:
+            None, None if fanout switch instance and port is not found
+            FanoutHost, Portname(str) if found
+    """
+    for _, fanout in fanout_switches.items():
+        if dut_port in fanout.host_to_fanout_port_map.keys():
```

Comment:
Cool (or weird) python-ism - you don't actually need to specify `keys()` here. `in` only checks the keys of the dictionary it's checking.

Reply:
Thanks! Changed.